src/components/CheckboxInput/CheckboxInput.tsx

```diff
@@ -31,32 +35,24 @@ export type CheckboxInputProps = CheckboxHTMLElementProps & {
 export const CheckboxInput = React.forwardRef<
   HTMLInputElement,
   CheckboxInputProps
->(({ checked, className, disabled, ...other }, ref) => {
-  // Make indeterminate checkbox visually match the colors of a checked state, but announce itself
-  // as "mixed" to screen readers
-  //
-  // However, https://html.spec.whatwg.org/multipage/input.html#checkbox-state-(type=checkbox)
-  // seems to consider `indeterminate` as a separate bit of information than `checked`. They can be
-  // in all 4 combinations (only 3 visual states, though).
-  //
-  // Should we make them separate props?
-  const checkedProps =
-    checked === 'indeterminate'
-      ? {
-          'aria-checked': 'mixed' as const,
-          checked: true,
```

Comment:
Previously we considered indeterminate checkboxes as "checked".

That's not correct, according to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#indeterminate_state_checkboxes.

Indeterminate checkboxes should be considered un-checked.

Reply:
Think `'indeterminate'` should be removed from the argtypes `checked` options